Lib/glyphsLib/__init__.py

```diff
@@ -56,22 +57,69 @@ def loads(value):
     return data
 
 
+def load_glyph_data(data=glyphdata_generated, glyphs_source=None, custom_glyph_xml=None):
+    """Load GlyphData.xml
+    If a custom GlyphData.xml exists merge it with the default GlyphData.xml
+    Two overrides are possible:
+        1. Reading from a custom GlyphData.xml
+        2. Manually changing the info in Glyphs app
+    Args:
+        data: pre-generated glyph data
+        glyphs_source: unpacked .glyphs object
+        custom_glyph_xml: Path to a custom GlyphData.xml file
+    """
+    glyph_dict = data.DEFAULT_GLYPH_DICT.copy()
+    if custom_glyph_xml:
+        custom_glyphs = {}
+        lines = None
+        with open(custom_glyph_xml, 'r') as f:
+            lines = f.read()
+        if lines:
+            custom_glyphs = fetch_all_glyphs(paths=(lines,))
```

Comment:
this `fetch_all_glyphs` function is no longer "fetching" anything, as in downloading something from a some remote server. And that `paths` argument is now being used to pass on the data itself (the string content), rather than the filesystem location where the containing file is.